public/config.js

```diff
@@ -0,0 +1,11 @@
+// Uncomment below lines to provide runtime configuration.
+//let CONFIG = {
+//    APP_NAME: "Logion Wallet",
+//    availableNodes: [
+//        {
+//            name: "Logion Node",
+//            peerId: "12D3KooWBmAwcd4PJNJvfV89HwE48nwkRmAgo8Vy3uQEyNNHBox2",
+//            socket: "ws://localhost:9944"
+//        }
+//    ]
+//};
```

Comment:
Looks like you need `CONFIG` to be defined for the tests (see failing test `UserRouter.test.tsx`)
```suggestion
let CONFIG = {
//    APP_NAME: "Logion Wallet",
//    availableNodes: [
//        {
//            name: "Logion Node",
//            peerId: "12D3KooWBmAwcd4PJNJvfV89HwE48nwkRmAgo8Vy3uQEyNNHBox2",
//            socket: "ws://localhost:9944"
//        }
//    ]
};
```